lib/config/db/migrations_performer.js

```diff
@@ -0,0 +1,50 @@
+import path from 'path';
+import fs from 'fs';
+import dotenv from 'dotenv';
+import { Migration } from 'mongration';
+import Q from 'q';
+import config from '../config';
+
+dotenv.config();
+
+const migrationConfig = {
+  hosts: config.get('MONGODB_HOST'),
+  db: config.get('MONGODB_DATABASE'),
+  user: config.get('MONGODB_USER'),
+  password: config.get('MONGODB_PASSWORD'),
+  migrationCollection: 'migrationversion',
```

Comment:
maybe this could also be put to a const in config? This way you could perform migrations for different branches, having a separate collection for each branch. Another way would be to put the logic into getAllMigrationScripts function given that you have some field indicating which branch this migration belongs to (one collection in this case).